bin/workloads/sql/scan/hadoop/run.sh

```diff
@@ -25,15 +25,31 @@ show_bannar start
 
 ensure_hivebench_release
 
-cp ${HIVEBENCH_TEMPLATE}/bin/hive $HIVE_HOME/bin
-
 # path check
 rmr_hdfs $OUTPUT_HDFS
 
 # prepare SQL
 HIVEBENCH_SQL_FILE=${WORKLOAD_RESULT_FOLDER}/rankings_uservisits_scan.hive
 prepare_sql_scan ${HIVEBENCH_SQL_FILE}
 
+if [[ $HADOOP_HOME =~ "3.2" || $HADOOP_HOME =~ "3.1" ]];then
+    echo " replace guava jar nad create metada schema"
+    # replace guava jar
+    rm -rf $HIVE_HOME/lib/guava-19.0.jar
+    cp ${HIVEBENCH_TEMPLATE}/lib/guava-27.0.1-jre.jar $HIVE_HOME/lib
+    # create metada schema
+    rm -rf $HIVE_HOME/../metastore_db
+    echo "$HIVE_HOME/bin/schematool -initSchema -dbType derby"
+    $HIVE_HOME/bin/schematool -initSchema -dbType derby
+elif [[ $HADOOP_HOME =~ "3.0" ]];then
+    echo " create metada schema"
+    # create metada schema
+    rm -rf $HIVE_HOME/../metastore_db
+    echo "$HIVE_HOME/bin/schematool -initSchema -dbType derby"
+    $HIVE_HOME/bin/schematool -initSchema -dbType derby
+fi
+
```

Comment:
We should put line 35-50 to  hadoopbench/sql/hive_template/bin/hive

Reply:
@gczsjdy  it's done.

Reply:
Great